### PR TITLE
Update proto to v0.8.0 release

### PR DESCRIPTION
**Pull request by MikeGoldsmith**
Updates the opentelemetry-proto submodule to v0.8.0 tag and re-generate proto files.

### CODE REVIEW

File: otlp/go.mod [hunk 1]
```diff
@@ -1,10 +1,10 @@
 module go.opentelemetry.io/proto/otlp
 
-go 1.15
+go 1.14
```
Comment (MrAlias) [inline otlp/go.mod:3]: Is this a needed change? I would like to maintain compatibility with the same minimum version supported by opentelemetry-go if possible.

Comment (MikeGoldsmith) [reply otlp/go.mod:3]: I believe opentelemetry-go targets [1.14](https://github.com/open-telemetry/opentelemetry-go/blob/main/go.mod#L3). This change brings it in line with the SDK.

Comment (MrAlias) [reply otlp/go.mod:3]: Oh shoot, you're right. I misread the diff 👍

**Review by MrAlias — APPROVED** [review]

**Comment by MikeGoldsmith** [comment]
@bogdandrutu Could you take a look too please? I'm unsure who're the approvers / maintainers for this project.

(I'm happy to help manage this project)

**Review by tigrannajaryan — APPROVED** [review]
The updated git submodule LGTM.
The generated files are hard to review so it would be great if we could make https://github.com/open-telemetry/opentelemetry-proto-go/issues/6 and https://github.com/open-telemetry/opentelemetry-proto-go/issues/4

**Comment by MikeGoldsmith** [comment]
@bogdandrutu ping 😄

**Comment by MikeGoldsmith** [comment]
Please could someone with write access to this repo create the v0.8.0 tag? Thanks.

**Pull request merged**